react-hs-examples/redux/stackjs.yaml

```diff
@@ -0,0 +1,17 @@
+resolver: lts-7.19
```

Comment:
we're using the more recent lts-8.11 ghcjs everywhere else.  any reason you install a different one here?  (the drawback is that it'll take a lot longer to build for everybody.  plus there are bug fixes missing from ghcjs in this particular example only.)